nifi-toolkit/nifi-property-encryptor-tool/src/main/java/org/apache/nifi/util/console/utils/SchemeCandidates.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.util.console.utils;
+
+import org.apache.nifi.properties.scheme.StandardProtectionSchemeResolver;
+
+import java.util.ArrayList;
+
+public class SchemeCandidates extends ArrayList<String> {
+    SchemeCandidates() {
+        super(new StandardProtectionSchemeResolver().getSupportedProtectionSchemes());
```

Comment:
Instead of returning the internal enum names, we should return the values of `ProtectionScheme.getPath()`. Those are the values used in the configuration files, and this is a good opportunity to switch the public arguments.

Reply:
Is there an existing way to enumerate all the possible ProtectionScheme paths available at run time? This SchemeCandidates class is used to present all available options to the user in the CLI and does validation checking on the user selection of the scheme

Reply:
There is no existing way to enumerate the Protection Scheme paths, so streaming the array of enum values and mapping to `getPath` in this class should work.